lib/dns.js

```diff
@@ -31,7 +72,7 @@ Client.prototype.query = function query(type, host, cb) {
     ra: true
   });
 
-  if (host[host.length - 1] !== '.')
+  if (host[ host.length - 1 ] !== '.')
```

Comment:
Hm, seems it was an issue with jscs 1.10 https://github.com/jscs-dev/node-jscs/issues/923.

After upgrading my jscs to 1.11.3, the "space" problem has disappeared. I'll update the pull request here. However, I still see style errors for bn.js:

```bash
~/Code/bn.js on master
$ jscs --version
1.11.3
~/Code/bn.js on master
$ jscs -c .jscsrc .
Line must be at most 80 characters at ./benchmarks/index.js :
    69 |var b5 = new sjcl(b1.toString(16));
    70 |
    71 |var a6 = new BigInteger('012345678901234567890123456789012345678901234567890', 10);
-------------------------------------------------------------------------------------------^
    72 |var b6 = new BigInteger('213509123601923760129376102397651203958123402314875', 10);
    73 |

...

Extra quotes for key at ./benchmarks/index.js :
   321 |    am1.redInvm();
   322 |  },
   323 |  'sjcl': function() {
----------^
   324 |    am5.inverseMod(prime5);
   325 |  }

Extra quotes for key at ./benchmarks/index.js :
   330 |    a1.gcd(b1);
   331 |  },
   332 |  'bigi': function() {
----------^
   333 |    a4.gcd(b4);
   334 |  },

Extra comma following the final element of an array or object literal at ./benchmarks/index.js :
   332 |  'bigi': function() {
   333 |    a4.gcd(b4);
   334 |  },
------------^
   335 |});
   336 |


48 code style errors found.
```